src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -486,7 +486,9 @@ public boolean deleteTestDataset(UUID id) throws Exception {
   }
 
   public boolean deleteTestProfile(UUID id) throws Exception {
-    MvcResult result = mvc.perform(delete("/api/resources/v1/profiles/" + id)).andReturn();
+    MvcResult result =
+        mvc.perform(delete("/api/resources/v1/profiles/" + id + "?deleteCloudResources=true"))
```

Comment:
The mvc APIs provide templating so you don't need to use string concatenation.
```suggestion
        mvc.perform(delete("/api/resources/v1/profiles/{id}?deleteCloudResources=true", id))
```